examples/http/build.gradle

```diff
@@ -7,9 +7,11 @@ ext.moduleName = "io.opentelemetry.examples.http"
 
 dependencies {
     compile("io.opentelemetry:opentelemetry-api")
-    compile("io.opentelemetry:opentelemetry-semconv:${openTelemetryAlphaVersion}")
     compile("io.opentelemetry:opentelemetry-sdk")
-    compile("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:${openTelemetryAlphaVersion}")
     compile("io.opentelemetry:opentelemetry-exporter-logging")
+
+    //alpha modules
+    compile("io.opentelemetry:opentelemetry-semconv")
+    compile("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
     compile "io.grpc:grpc-context"
```

Comment:
Grpc is not alpha. Maybe a new line and comment third-party?

Reply:
Also why is grpc context here?

Reply:
great question...no clue about that! Probably an old remnant. I'll work on cleaning it up some more.

Reply:
ok, cleaned up!